docs/variants/msi_z690/firmware-update.md

```diff
@@ -0,0 +1,32 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. The update process may be different, depending on which
+firmware version is currently installed on your device. The currently installed
+firmware version can be checked with the following command in a Linux
+environment:
+
+```bash
+sudo dmidecode -t bios | grep Version
+```
+
+Alternatively, it can be checked in the `BIOS Setup Menu`.
+
+## Version v1.0.0 or newer
```

Comment:
Whether the correctness of these commands has been confirmed by testing them on the platform?